example/build.gradle

```diff
@@ -39,13 +39,15 @@ android {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt')
 
-            signingConfig signingConfigs.release
+            if (file(releaseKeyStoreFileName).exists() && file(releasePropFileName).exists()) {
+                signingConfig signingConfigs.release
```

Comment:
I think I prefer the original fail-fast setup here. An unsigned release apk is not particularly useful as far as I know. Plus, building the debug variant should be fine for most library users - the release configuration only exists for me to update the store build. Please revert this commit!